releases/v1.5.0-rc.toml

```diff
@@ -52,6 +53,7 @@ brings support for the Node Resource Interface (NRI).
 * **Optimize LCOW snapshotter use of scratch layers** [#4643](https://github.com/containerd/containerd/pull/4643)
 
 #### CRI
```

Comment:
We enabled ocicrypt by default https://github.com/containerd/containerd/pull/5135 https://github.com/containerd/containerd/pull/5308